backend/api/handlers/application/PostTutorApplicationHandler.go

```diff
@@ -25,25 +25,17 @@ func PostTutorApplicationHandler(c *gin.Context) {
 	}
 
 	//validate max char lengths
-	err = ensureMaxChar(50, request.Guardian2FirstN)
-	err = ensureMaxChar(50, request.Guardian2LastN)
-	err = ensureMaxChar(20, request.Guardian2Phone)
-	err = ensureMaxChar(255, request.Guardian2Email)
-
 	err = ensureMaxChar(50, request.EmergencyConName)
 	err = ensureMaxChar(255, request.EmergencyConRelation)
 	err = ensureMaxChar(20, request.EmergencyConPhone)
 
-	err = ensureMaxChar(50, request.WhatSSemester)
 	err = ensureMaxChar(50, request.ChildCurrentSchool)
 	err = ensureMaxChar(255, request.ListLanguagesSpoken)
 	err = ensureMaxChar(511, request.ReceivedSpecialEd)
 	err = ensureMaxChar(511, request.ListChallenges)
 	err = ensureMaxChar(255, request.HowLongConcerned)
 	err = ensureMaxChar(511, request.DescribeHopes)
 	err = ensureMaxChar(255, request.ChildAllergyMeds)
-	err = ensureMaxChar(511, request.MiscInfo)
-	err = ensureMaxChar(255, request.HearAboutLitLab)
 
 	if err != nil {
```

Comment:
the only err that actually gets checked here is the last one

Reply:
oof, lol good point